dev/crypto-notes.md

```diff
@@ -0,0 +1,35 @@
+## Important notes
+
+### Key-pair generation
+
+When compared to gpg without hardware-based (P)RNG, IPFS generates key-pair
+alarmingly fast: it takes ipfs about 1 minute to generate 4096-bit
```

Comment:
this note will get outdated quick and not updated. dont say `IPFS generates key-pairs alarmingly fast`, say `At the time of writing this note, it seems that `ipfs init` generates key-pairs much faster than gpg.`